hazelcast/src/main/java/com/hazelcast/internal/management/TimedMemberState.java

```diff
@@ -48,6 +48,7 @@
     boolean socketInterceptorEnabled;
     boolean scriptingEnabled;
     boolean consoleEnabled;
+    boolean mcDataAccessEnabled;
```

Comment:
Why do we have that `mc` prefix here?

Reply:
just to be explicit on what data access is enabled/disabled